smack-core/src/test/java/org/jivesoftware/smack/test/util/CharsequenceEquals.java

```diff
@@ -1,47 +0,0 @@
-/**
```

Comment:
Can you revert this change, as it is already taken care for in #209?

Reply:
Yes sure, I will do it.